COMPILING

```diff
@@ -203,3 +203,20 @@ To work with Eclipse, do the following:
 6) Select Project -> Build All
 
 
+CODE COVERAGE
+-------------
+
+Code coverage metrics are provided using gcov and lcov. Ensure that you
+have installed lcov from http://ltp.sourceforge.net/coverage/lcov.php
+note for ubuntu lcov is available in the standard apt-get repos.
+
+To get coverage metrics run the following script from the regression
+directory:
+
+  getCoverage.sh
```

Comment:
Thanks for the review, I have made this change, I have also attempted to squash the commits, please let me know if there is anything else.